source/modules/MEI.shared.xml

```diff
@@ -3559,6 +3565,40 @@
         C to C♯ and one from C to D♭.</p>
     </remarks>
   </classSpec>
+  <classSpec ident="att.tuning.log" module="MEI.shared" type="atts">
+    <desc xml:lang="en">Logical domain attributes for tuning.</desc>
+    <attList>
+      <attDef ident="tuning.standard" usage="opt">
+        <desc xml:lang="en">Describes the tuning standard used.</desc>
+        <datatype>
+          <rng:data type="NMTOKEN"/>
+        </datatype>
+        <valList type="semi">
```

Comment:
Aren't these values the same as `data.COURSETUNING`?

Reply:
Well spotted. We have now deduplicated this code.